docs.md

```diff
@@ -0,0 +1,40 @@
+## General Moments
+
+### Everything is a Knowledge Item. 
+#### KnowledgeItem 
+KnowledgeItem - abstract class, that is parental for all other items
+**Unique fields: _title, creation_time,  id: **
+
+
+### Information Classes 
```

Comment:
I'm not sure we need to list all subclasses here - it will be hard to maintain when new classes are added
Maybe add a notice that these are examples of such classes?